config/registry-config.yaml

```diff
@@ -0,0 +1,41 @@
+# https://docs.docker.com/registry/configuration/
```

Comment:
Where is this file used? it seems we're not passing its path as argument anywhere.

Reply:
What's the answer to this question?  Can we remove it?  If it's going to be used it would be in the docker-compose file below, but you have it commented out.

Reply:
I was unsure of the volume naming convention for it, and it would seem that the default configuration is adequate for what we have so far. I'm not sure this config file is needed?

Reply:
The only thing we really need from this is to configure auth.  Before we can use this for real, we need to have auth.  In a follow-up we need:

- GitHub-based auth, https://github.com/cesanta/docker_auth/blob/main/examples/reference.yml#L122-L163
- Probably a Redis instance added to `docker-compose.yml` and configured to cache the tokens

Reply:
@TDDR delete this file until we need it.